native/effektio/Cargo.toml

```diff
@@ -60,8 +60,13 @@ wasm-bindgen-futures = "0.4.28"
 [target.'cfg(target_os = "android")'.dependencies]
 tracing-android = "0.2"
 android_logger = "0.10.1"
-tokio = { version = "1", features = ["rt-multi-thread", "parking_lot"] }
+tokio = { version = "1", features = ["rt-multi-thread", "parking_lot", "sync"] }
 
 [lib]
 name = "effektio"
 crate-type = ["cdylib", "staticlib"]
+
+[dev-dependencies]
+tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
+tokio-test = "*"
+zenv = "0.8.0"
```

Comment:
```suggestion
zenv = "0.8.0"
matrix-sdk-crypto = { git = "https://github.com/matrix-org/matrix-rust-sdk/" }

```

Reply:
Thank you.
This code is important for me.